codalab/worker/bundle_service_client.py

```diff
@@ -43,6 +43,15 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
+def ensure_str(error):
+    '''
+    Ensure the type of urllib.error.HTTPError response to be string
+    :param error: urlbli.error.HTTPError response
+    :return: HTTPError response in string type
+    '''
+    return error if isinstance(error, str) else error.decode()
```

Comment:
Why would the `decode()` work if the underlying data is `bytes`?

Reply:
`bytes.deocde()` will change to `string`